packages/lesswrong/server/vulcan-lib/apollo-ssr/renderPage.tsx

```diff
@@ -107,7 +110,7 @@ export const renderWithCache = async (req: Request, res: Response, user: DbUser|
   // Doing this so we can show dynamic latest posts list with varying HN decay parameters based on visit frequency (see useractivities/cron.ts).
   const showDynamicFrontpage = !!lastVisitedFrontpage && visitorGetsDynamicFrontpage(user) && url === "/";
   
-  if ((!isHealthCheck && (user || isExcludedFromPageCache(url, abTestGroups))) || isSlackBot || showDynamicFrontpage) {
+  if ((!isHealthCheck && (user || isExcludedFromPageCache(url))) || isSlackBot || showDynamicFrontpage) {
```

Comment:
I'll handle the `isSlackBot` check with an edge function on CloudFront (generate a header that is used as part of the cache key for special cases like this)